src/libs/ApiUtils.ts

```diff
@@ -38,12 +38,14 @@ function getApiRoot(request?: Request): string {
     const shouldUseSecure = request?.shouldUseSecure ?? false;
 
     if (shouldUseStagingServer) {
-        if (CONFIG.IS_USING_WEB_PROXY) {
+        if (CONFIG.IS_USING_WEB_PROXY && !request?.shouldSkipWebProxy) {
```

Comment:
> since we mainly use the proxy to avoid CORS errors and (I presume) there is no problem with CORS errors when using a WebView this could be updated to something like request.isForWebView.

This problem doesn't only apply to WebViews, it is also there in "web" when the URL opens in a new tab. The URL created when we use the proxy is `/CommandName?xxx`, it misses the domain, so then the function `Link.openLink` misses that it needs to get a short lived auth token for it.

Reply:
`isForWebViewOrNewTab` 🙃

Reply:
or maybe... `requiresShortLivedAuthToken` 😬

Reply:
yeah, I like that better.